README.rst

```diff
@@ -381,8 +381,9 @@ The following flags are accepted:
 | :flag:`-go_prefix example.com/repo`                          |                                        |
 +--------------------------------------------------------------+----------------------------------------+
 | A prefix of import paths for libraries in the repository that corresponds to                          |
-| the repository root. Gazelle infers this from the ``go_prefix`` rule in the                           |
-| root BUILD.bazel file, if it exists. If not, this option is mandatory.                                |
+| the repository root. Gazelle infers this from the ``# gazelle:prefix`` directive                      |
```

Comment:
It would be more accurate to say that setting `-go_prefix` on the command line is equivalent to putting a `# gazelle:prefix` directive in the root build file. The `prefix` attribute on the `gazelle` rule actually causes `gazelle` to be invoked with this flag.

Reply:
Ah, I think I get it.  It's updated.  Does it makes sense now?

Reply:
That works. Thanks!